nominees/ekshop.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "ekShop",
+  "aliases": [
+    ""
+  ],
+  "description": "ekShop is a fully open-source eCommerce platform that is customizable and configurable to your needs. It is completely free, adaptable and open to be supported by a worldwide community of volunteers and contributors. It’s free and open source nature allows users to maintain complete control of the data content and modify it as they wish and according to their needs. Our intended goal is to allow any potential entrepreneur to quickly get up and running with an online platform to sell goods online.",
+  "website": "https://ekshop.gov.bd/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/a2i-dpg/ekShop/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 1,
+      "evidenceText": "ekShop is a scalable open-source eCommerce solution that provides the power of harnessing online marketplace and businesses into the hands of everyday folks. ekShop allows people to bring forth sustainable livelihoods by engaging entrepreneurs, merchants, and employees that help the business operate. As a result, ekShop, when implemented correctly, can uplift communities and populations out of poverty."
+    },
+    {
+      "SDGNumber": 5,
+      "evidenceText": "ekShop has strict policies for preventing all forms of gender discrimination by offering equal opportunity to every gender, race, and ethnicity to participate in its platform - both in the capacity of buyer and seller. Women, in particular, can find the means of selling their creations and handicrafts through the platform and gain reach, unlike other means before. "
```

Comment:
I'd like to see some evidence of how an e-commerce platform prevents gender discrimination. Can we ask for these policies mentioned in the comment? This would make it easier to validate this claim. If their claim is that gender discrimination is avoided on the basis of it being public and available under an open source license, I would think not, especially considering the open secret that open source has a [worse gender skew](https://arxiv.org/abs/2105.08777) than the technology industry as a whole.

Reply:
Resolved in commit 6cb896137447acf8bbb26f5cdc9691d7001fd7e3.